app/javascript/tests/patient/Patient.test.js

```diff
@@ -84,6 +96,14 @@ describe('Patient', () => {
     });
   });
 
+  it('Properly renders contact information section when patient is a minor', () => {
+    const wrapper = shallow(<Patient details={mockPatient5} hoh={mockPatient1} collapse={true} edit_mode={false} jurisdiction_path="USA, State 1, County 2" />);
+    expect(wrapper.find('#contact-information').find('.text-danger').exists()).toBeTruthy();
+    expect(wrapper.find('#contact-information').find('.text-danger').text()).toEqual('Monitoree is a minor. ');
+    expect(wrapper.find('#dependent-hoh-link').exists()).toBeTruthy();
+    expect(wrapper.find('#dependent-hoh-link').text()).toEqual(mockPatient1.first_name + ' ' + mockPatient1.middle_name + ' ' + mockPatient1.last_name);
```

Comment:
you should add checks here for the rest of the contact info section (see above test)